docs/content/rules/sort-classes.mdx

```diff
@@ -215,146 +215,194 @@ Allows you to use comments to separate the class members into logical groups. Th
 
 Allows you to specify a list of class member groups for sorting. Groups help organize class members into categories, prioritizing them during sorting. Multiple groups can be combined to achieve the desired sorting order.
 
-There are a lot of predefined groups.
-
-Predefined Groups:
-
-- `'index-signature'` — Index signatures, which define the types of keys and values in an object.
-- `'protected-decorated-accessor-property'` — Protected accessor properties with decorators.
-- `'private-decorated-accessor-property'` — Private accessor properties with decorators.
-- `'decorated-accessor-property'` — Accessor properties with decorators.
-- `'protected-decorated-property'` — Protected properties with decorators.
-- `'private-decorated-property'` — Private properties with decorators.
-- `'decorated-property'` — Properties with decorators.
-- `'protected-property'` — Protected properties.
-- `'private-property'` — Private properties.
-- `'static-property'` — Static properties.
-- `'property'` — Regular properties.
-- `'constructor'` — Constructor method.
-- `'protected-method'` — Protected methods.
-- `'private-method'` — Private methods.
-- `'static-protected-method'` — Static protected methods.
-- `'static-private-method'` — Static private methods.
-- `'static-method'` — Static methods.
-- `'decorated-method'` — Methods with decorators.
-- `'decorated-get-method'` — Getter methods with decorators.
-- `'decorated-set-method'` — Setter methods with decorators.
-- `'get-method'` — Getter methods.
-- `'set-method'` — Setter methods.
-- `'method'` — Regular methods.
-- `'unknown'` — Members that don’t fit into any other group.
+Predefined groups are characterized by a single selector and potentially multiple modifiers. You may enter modifiers in any order, but the selector must always come at the end.
 
-Example:
+#### Methods
+- Selectors: `constructor`, `get-method`, `set-method`, `method`.
+- Modifiers: `abstract`, `decorated`, `override`, `protected`, `private`, `public`, `static`.
+- Example: `protected-abstract-override-method` or `static-get-method`.
+
+#### Accessors
```

Comment:
Maybe merge with the previous section since it's the same usage:
```md
#### Methods and accessors
- Selectors: `get-method`, `set-method`, `method`, `accessor-property`.
 - Modifiers: `abstract`, `decorated`, `override`, `protected`, `private`, `public`, `static`.
 - Example: `private-static-accessor-property`.
 ```